src/components/ZclCustomZclView.vue

```diff
@@ -87,21 +90,36 @@ import { Notify } from 'quasar'
 export default {
   mixins: [CommonMixin],
   methods: {
+    fileSelected(event){
+      this.packageToLoad = event.target.files[0]
+      this.loadNewPackage()
+    },
+    browseForFile() {
+      if(this.standaloneMode() && window.navigator.platform.includes('Linux')){
```

Comment:
when you try to add file in linux standalone it will send a path of the file. but if you try to do that in another ways ( windows browser etc ) it will be upload the file in to the project directory ( user-files ).

Reply:
Add this description above the if case. Does this work right for all use cases when (standalone && platformIsNotLinux) ?

Reply:
I will add somthing for mac too so we can pick file in every mode in every platform